README.rst

```diff
@@ -20,7 +20,10 @@ Connecting
 
     # Public apps (OAuth)
     # Access_token is optional, if you don't have one you can use oauth_fetch_token (see below)
+    # For connecting to the v2 api:
     api = bigcommerce.api.BigcommerceApi(client_id='', store_hash='', access_token='')
+    # For connecting to the v3 api:
+    api = bigcommerce.api.BigcommerceApi(client_id='', store_hash='', access_token='', api_path='/stores/{}/v3/{}'))
```

Comment:
you're overriding `api` variable, remove the previously line if not required

Reply:
This is the README file and both version are supported, so this lines are useful about how to choose one or another.